conformance/tests/gateway-observed-generation-bump.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway-observed-generation-bump
```

Comment:
Any way to use an existing Gateway for this?

Reply:
I would prefer to keep tests isolated - I don't want to change the base gateway if other tests assume it's static.